bin/deploy

```diff
@@ -56,23 +60,32 @@ cd "$LSSTSW"
 
 test -f "$LSSTSW/miniconda/.deployed" || ( # Anaconda
     cd sources
+
+    if [[ $USE_PYTHON3 == true ]]; then
+        PYVER_PREFIX=3
+        MINICONDA_VERSION=${MINICONDA3_VERSION}
+    else
+        PYVER_PREFIX=2
+        MINICONDA _VERSION=${MINICONDA2_VERSION}
```

Comment:
THat's confusing because I fixed that last week (otherwise Travis would not work for lsst_build).